modules/electrophysiology_browser/README.md

```diff
@@ -0,0 +1,35 @@
+# Electrophysiology Browser
```

Comment:
@christinerogers let me know if you have any feedback on the README.md (or other files too ;))

Reply:
This looks like a good first draft. I'll think later about what it might be missing. 
The only change I can suggest (and I'll make this commit myself) -- change "Viewing" to "Visualization"

Reply:
We should also add: (this is a note to self) 
- [ ] Download -- all the different great things you can download! (and who has permission to do it?  Site-based?

Reply:
You are so right!! I totally forgot... I will add it in the next commit.